.travis.yml

```diff
@@ -20,15 +20,14 @@ matrix:
   allow_failures:
     - php: nightly
 
-before_install:
-  - composer global config repositories.bin path $PWD
-  - composer global require bamarni/composer-bin-plugin:dev-master
-
 install:
   - composer install
+  - composer global require "bamarni/composer-bin-plugin:dev-master"
```

Comment:
makes sense, so I'm just missing the `composer global config repositories.bin path $PWD` right?